packages/core-data/src/entities.js

```diff
@@ -57,10 +57,10 @@ export const defaultEntities = [
 	{
 		name: 'widgetArea',
 		kind: 'root',
-		baseURL: '/__experimental/widget-areas',
+		baseURL: '/__experimental/sidebars',
 		plural: 'widgetAreas',
 		transientEdits: { blocks: true },
-		label: __( 'Widget area' ),
+		label: __( 'Widget areas' ),
 	},
```

Comment:
I'm not convinced about this part, `sidebar` is a terribly unintuitive name for a widget area. I understand it's the standard name used in core too and so I'm torn.

Reply:
I renamed just the entity and retained the `widgetArea` name in selectors, resolvers etc. Happy to discuss it further in a  follow-up issue.

Reply:
Agree that it's a regrettable name, but I'd personally prefer that the combined codebase (Core+Gutenberg) has one regrettable name for something instead of two names for something.